scripts/delete_old_version_testing_branches.sh

```diff
@@ -3,9 +3,35 @@
 set -eo pipefail
 
 today=$(date +%Y%m%d)
-echo "today: $today"
-git fetch --prune
-version_testing_branches=$(git branch -r | grep -E '(version-testing-[0-9]{8})' | grep -v "$today" | awk -F'/' '{print $2}')
-echo "branches to be removed:"
-echo "$version_testing_branches"
-echo "$version_testing_branches" | xargs -I {} git push origin --delete {}
+echo "Today: $today"
+if [ "$CI" = true ]; then
+    unshallow_flag="--unshallow"
+fi
+git fetch $unshallow_flag --force
+echo "Gathering version testing branches..."
+version_testing_branches=$(git branch -r | grep -E '(version-testing-[0-9]{8})' || echo "")
```

Comment:
Could you print:
1. all branches
2. branches that match version testing

It could help in future debugging if this segment fails somehow

Reply:
printing *all* branches is too much, but for the rest, done